### PR TITLE
LPS-120376 Created a merged single sql in portal | 2

### PR DESCRIPTION
**Pull request by dantewang**

### CODE REVIEW

**Comment by dantewang** [comment]
ci:test:sf

**Comment by dantewang** [comment]
ci:test:relevant

**Comment by dantewang** [comment]
CI seems to be broken at the moment.

File: benchmarks/build.xml [hunk 1]
```diff
@@ -80,7 +80,17 @@
 			<path refid="project.classpath" />
 		</path>
 
-		<property name="sample.sql.properties.file" value="benchmarks.properties" />
+		<available file="benchmarks.${user.name}.properties" property="user.properites.avaiable" />
```
Comment (tinatian) [inline benchmarks/build.xml:83]: @slnn 
You include build.properties like this:
`<property file="build.${user.name}.properties" />
<property file="build.properties" />`
 Why you include benchmark.properties use the same way?

FYI @dantewang 

Tina.

Comment (slnn) [reply benchmarks/build.xml:83]: Resend in https://github.com/dantewang/liferay-portal/pull/1774

<property file="build.${user.name}.properties" /> <property file="build.properties" />

I set all these import related properties in build.properties file originally, I forgot to remove these unused includes after moving to benchmark.properties.

**Comment by tinatian** [comment]
close as we have talked, please resend.

**Pull request closed without merging**